pom.xml

```diff
@@ -366,7 +366,7 @@
         <extensions>true</extensions>
         <configuration>
           <instructions>
-            <_exportcontents>co.cask.hydrator.plugin.*</_exportcontents>
+            <_exportcontents>io.cdap.plugin.*</_exportcontents>
```

Comment:
please also move all classes to io.cdap.plugin.dynamo and change this to io.cdap.plugin.dynamo.*.

We need to avoid creating plugins that export the same packages.

Reply:
moved classes to io.cdap.plugin.dynamo.source to avoid conflicts with Dynamodb Sink Plugin